site/.vitepress/config.ts

```diff
@@ -126,6 +126,14 @@ export default defineConfig({
             text: "How to Transfer Ownership of a Smart Account",
             link: "/transferring-ownership",
           },
+          {
+            text: "How to Fetch a Smart Account's NFTs",
+            link: "/enhanced-apis-nft",
+          },
+          {
+            text: "How to Fetch a Smart Account's ERC-20 Tokens",
+            link: "/enhanced-apis-token",
+          },
```

Comment:
can you create an isolated subsection inside guides for Enhanced?

That's what I am doing for gas sponsorship doc(s) https://github.com/alchemyplatform/aa-sdk/pull/225